imagelab/DisplayImage.java

```diff
@@ -94,5 +100,17 @@ public DisplayImage(ImgProvider imp, String title, boolean slow) {
         setJMenuBar(myMenuBar);
         setVisible(true);
     }
+    
+    public void remove(){
+        getContentPane().remove(pane);
+    }   
+    
+     public void synchronize(int w, int h, int[] pixs){
+        source = new MemoryImageSource(w, h, pixs, 0, w);
```

Comment:
I have removed the use of this instance variable and just used a local variable. This will be provided in a new commit and PR.